tests/test_move.py

```diff
@@ -60,3 +63,91 @@ def test_move_dir(self):
         if self.preserve_time:
             dst_file2_info = dst_fs.getinfo("bar/baz.txt", namespaces)
             self.assertEqual(dst_file2_info.modified, src_file2_info.modified)
+
+
+class TestMove(unittest.TestCase):
+    def test_move_file(self):
+        with open_fs("temp://") as temp:
+            syspath = temp.getsyspath("/")
+            a = open_fs(syspath)
+            a.makedir("dir")
+            b = open_fs(join(syspath, "dir"))
+            b.writetext("file.txt", "Content")
+            fs.move.move_file(b, "file.txt", a, "here.txt")
+            self.assertEqual(a.readtext("here.txt"), "Content")
+            self.assertFalse(b.exists("file.txt"))
+
+    def test_move_file_different_mems(self):
+        with open_fs("mem://") as src, open_fs("mem://") as dst:
+            src.writetext("source.txt", "Source")
+            fs.move.move_file(src, "source.txt", dst, "dest.txt")
+            self.assertFalse(src.exists("source.txt"))
+            self.assertEqual(dst.readtext("dest.txt"), "Source")
+
+    def test_move_file_fs_urls(self):
+        # create a temp dir to work on
+        with open_fs("temp://") as tmp:
+            path = tmp.getsyspath("/")
+
+            tmp.writetext("file.txt", "Content")
+            tmp.makedir("subdir")
+            fs.move.move_file(
+                "osfs://" + path,
+                "file.txt",
+                "osfs://" + join(path, "subdir"),
+                "file.txt",
+            )
+
+            self.assertFalse(tmp.exists("file.txt"))
+            self.assertEqual(tmp.readtext("subdir/file.txt"), "Content")
+
+        with open_fs("mem://") as src, open_fs("mem://") as dst:
+            src.writetext("source.txt", "Source")
+            fs.move.move_file(src, "source.txt", dst, "dest.txt")
+            self.assertFalse(src.exists("source.txt"))
+            self.assertEqual(dst.readtext("dest.txt"), "Source")
```

Comment:
This seems to be a duplicate of the code in `test_move_file_different_mems` ??

Reply:
You've marked this as resolved, but I don't think this chunk of code belongs in `test_move_file_fs_urls` ?

Reply:
Totally missed that!